boards/arm/nucleo_g431kb/nucleo_g431kb.dts

```diff
@@ -0,0 +1,131 @@
+/*
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <st/g4/stm32g431Xb.dtsi>
+#include <st/g4/stm32g431k(6-8-b)tx-pinctrl.dtsi>
+#include "arduino_nano_r3_connector.dtsi"
+
+/ {
+	model = "STMicroelectronics STM32G431KB-NUCLEO board";
+	compatible = "st,stm32g431kb-nucleo";
+
+	chosen {
+		zephyr,console = &usart2;
+		zephyr,shell-uart = &usart2;
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+		zephyr,canbus = &can1;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		green_led: led_0 {
+			gpios = <&gpiob 8 GPIO_ACTIVE_HIGH>;
+			label = "User LD2";
+		};
+	};
+	aliases {
+		led0 = &green_led;
+	};
+
+};
+
+&clk_hsi {
+	status = "okay";
+};
+
+&pll {
+	div-m = <4>;
+	mul-n = <85>;
+      div-p = <7>;
+	div-q = <2>;
```

Comment:
Note that `div-p` and `div-q` are not mandatory if not required.

Reply:
div-p e div-q were deleted.